tox.ini

```diff
@@ -116,6 +116,7 @@ deps =
 commands =
     pytest -n auto \
         --ignore={[vars]tst_path}smoke \
+        --ignore={[vars]tst_path}integration \
```

Comment:
Not for this PR, but I think we definitely want to reorganise our tests and tox some time in 25.10, so that we're more "run these tests" and less "run all the tests except a,b,c,d".

Reply:
yep, that's for a separate effort